charts/values.yaml

```diff
@@ -144,97 +145,77 @@ additionalVolumeMounts: []
   ## NOTE : When adding new app servers to MarkLogic, the new app server ports must    ##
   ## also be added to the additionalPorts parameter to expose the ports via the        ##
   ## kubernetes service.                                                               ##
   #######################################################################################
-
   ## @param service.additionalPorts. Additional service ports 
-  ## Example:
-  ## - name: app1
-  ##   port: 8010 
-  ##   targetPort: 8010
-  ##   protocol: TCP
   additionalPorts: []
+  # - name: app1
+  #   port: 8010 
+  #   targetPort: 8010
+  #   protocol: TCP
 
 serviceAccount:
-  # Specifies whether a service account should be created
+  ## Specifies whether a service account should be created
   create: true
-  # Annotations to add to the service account
+  ## Annotations to add to the service account
   annotations: {}
-  # The name of the service account to use.
-  # If not set and create is true, a name is generated using the fullname template
+  ## The name of the service account to use.
+  ## If not set and create is true, a name is generated using the fullname template
   name: ""
 
-# Configure priority class for pods 
-# ref: https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/
+## Configure priority class for pods 
+## ref: https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/
 priorityClassName:  ""
 
-# Configure options for network policy
-# ref: https://kubernetes.io/docs/concepts/services-networking/network-policies
+## Configure options for network policy
+## ref: https://kubernetes.io/docs/concepts/services-networking/network-policies
 networkPolicy:
   enabled: false
   ## @param networkPolicy.customRules. Additional NetworkPolicy rules
   ## Note that all rules are OR-ed.
-  ## Example:
-  ## customRules:
-  ##   - matchLabels:
-  ##       - role: frontend
-  ##   - matchExpressions:
-  ##       - key: role
-  ##         operator: In
-  ##         values:
-  ##           - frontend
-  ##
   customRules: {}
+    # - matchLabels:
+    #     - role: frontend
+    # - matchExpressions:
+    #     - key: role
+    #       operator: In
+    #       values:
+    #         - frontend
   ## The endPort should be the last port exposed by an App Server
   ports:
     - port: 8000
       endPort: 8020
       protocol: TCP
     
-# Below are the security configurations for container, by default security will be enabled
+## Below are the security configurations for container, by default security will be enabled
 containerSecurityContext:
   enabled: true
   runAsUser: 1000
   runAsNonRoot: true
   allowPrivilegeEscalation: true
 
-# Below are the advanced configurations, please understand read the reference before you make changes
+## Below are the advanced configurations, please understand read the reference before you make changes
```

Comment:
"Below are the advanced configurations, please read the reference in detail before making changes"